.ci.yaml

```diff
@@ -1816,6 +1816,15 @@ targets:
         ["devicelab"]
     scheduler: luci
 
+  - name: Mac_ios animation_with_microtasks_perf_ios__timeline_summary
+    builder: Mac_ios animation_with_microtasks_perf_ios__timeline_summary
+    presubmit: false
+    bringup: true
```

Comment:
Can you add a comment linking to the github issue? It's helpful to know this is in bringup versus flaky

Reply:
Thanks, lgtm